relay/config.example.yml

```diff
@@ -10,3 +10,4 @@ processing:
     - {name: "bootstrap.servers", value: "kafka:9092"}
     - {name: "message.max.bytes", value: 50000000} #50MB or bust
   redis: redis://redis:6379
+  geoip_path: "/geoip/GeoLite2-City.mmdb"
```

Comment:
If we point these to a specific file, why do we need the `.empty` file?

Reply:
1. It's simpler for our users (operators) if we set a path in the default `*.example.*` config files, so that they don't have to set the path themselves in order to turn on ip geolocation. This is especially true since there are multiple similarly named files/dbs that MaxMind provides, which is a source of confusion/error.

1. Relay (and maybe Sentry?) hard-fail on startup if the file pointed to by `geoip_path` doesn't exist. So to satisfy (1) we need to have _some_ file there.

1. We want `geoip/GeoLite2-City.mmdb` itself to be gitignored, because we want to continually update it in production without cluttering git status.

1. By shipping an empty mmdb file and copying it into place if otherwise non-existent during install.sh, we keep git status clean (3) while allowing relay to start (2) with minimal confusion for users (1).